xep-0004.xml

```diff
@@ -295,6 +306,8 @@
       <li>One and only &lt;reported/&gt; element, which can be understood as a "table header" describing the data to follow.</li>
```

Comment:
while you're touching this section and creating a diff anyways, maybe add the missing "one" here so it can go in the same version bump? (in case this gets accepted)